examples/karpenter/main.tf

```diff
@@ -188,36 +194,34 @@ module "karpenter_launch_templates" {
   }
 }
 
-module "eks_blueprints_kubernetes_addons" {
-  source = "../../modules/kubernetes-addons"
+# module "eks_blueprints_kubernetes_addons" {
```

Comment:
Should we revert the commenting out of the values here?

Reply:
Done. Sorry I mistakenly pushed garbage along with my changes in last commit. I have amended it.